openapi.yaml

```diff
@@ -85,35 +166,71 @@ tags:
     ```
 
   externalDocs:
-    description: Full docs
+    description:
+      en: |
+        Full docs
+      fr: |
+        Docs complets 
     url: https://docs.ckan.org/en/2.8/api/
 security:
 - {}
 - internal_api_key: []
 paths:
   "/action/status_show":
     get:
-      summary: Get the CKAN version and a list of installed extensions
-      description: Use this API to check that you can reach the server and to check its capabilities.
+      summary: 
+        en: |
+          Get the CKAN version and a list of installed extensions
+        fr: |
+          Obtenir la version de CKAN et la liste des extensions installées. 
+      description: 
+        en: |
+          Use this API to check that you can reach the server and to check its capabilities.
+        fr: |
+          Utilisez cette API pour vous assurer que vous pouvez atteindre le serveur et pour vérifier ses capacités. 
       tags:
       - action
       responses:
         '200':
-          description: Returns the CKAN version and a list of installed extensions
+          description: 
+            en: |
+              Returns the CKAN version and a list of installed extensions
+            fr: |
+              Renvoie la version de CKAN et la liste des extensions installées. 
   "/action/package_list":
     get:
-      summary: Get a list of all package ids (datasets)
-      description: >
-        Note: our site uses the same value for the "name" and "id" fields of datasets
+      summary: 
+        en: |
+          Get a list of all package ids (datasets)
+        fr: |
+          Obtenir une liste de tous les identifiants des paquetages (jeux de données). 
+      description:
+        en: |
+          Note: our site uses the same value for the "name" and "id" fields of datasets
+        fr: |
+          Remarque : notre site utilise la même valeur pour les champs « nom » et « identifiant » des jeux de données 
```

Comment:
field names can't be translated, or maybe there's a way to provide both the field names and their translations in this text like `les champs « name » (nom) et « id » (identifiant)`? This is a question for the translation team.

Reply:
Applied recommendation